swagger/public_v1/api_spec.json

```diff
@@ -1108,6 +1189,97 @@
           }
         }
       }
+    },
+    "/recruitment_cycles/{year}/providers": {
+      "get": {
+        "summary": "Return providers",
+        "operationId": "public_api_v1_provider_index",
+        "tags": [
+          "provider"
+        ],
+        "parameters": [
+          {
+            "name": "year",
+            "in": "path",
+            "required": true,
+            "description": "The starting year of the recruitment cycle",
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "name": "filter",
+            "in": "query",
+            "style": "deepObject",
```

Comment:
![image](https://user-images.githubusercontent.com/470137/84121141-a6b02280-aa2e-11ea-9949-af17eb089e73.png)

Can we put back the description please.

Reply:
good spot. i will fix

Reply:
done, its now on the courses and providers endpoint